lib/modules/resumemodule/resumeresponse.py

```diff
@@ -0,0 +1,45 @@
+from lib.modules.databasemodule.database import database
+
+def resume_added():
+    """Returns the response for when a resume is successfully added"""
+
+    return "Resume added."
+
+def resume_not_added(error):
+    """Returns the response for when a resume is not successfully added"""
+
+    return "Resume not added: " + error
+
+def resume_removed():
+    """Returns the response for when a resume is successfully removed"""
+
+    return "Resume removed."
+
+def resume_not_removed(error):
+    """Returns the response for when a resume is not successfully removed"""
+
+    return "Resume not removed: " + error
+
+def list_resumes(db: database, id=None):
+
+    # Currently, we save the company ID, resume URL, and email in the database. The example says that 
+    # the author's name is shown... how will we do this? Also, where will we store the company ID?
```

Comment:
email should be connected to name via Slack. Lots of ways to solve this problem lmk if u want to brainstorm a few.